eclipse-theia/theia-blueprint
# WIP: Update to Theia 1.42.0

#### What it does
* Upgrades Theia to 1.42.0
* also upgrade dependencies
* update webpack configs, because manual changes have been integrated upstream by now

Contributed on behalf of STMicroelectronics

#### How to test
Build Blueprint in dev and production modes and check if everything is still working

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
